src/library/scala/concurrent/impl/Promise.scala

```diff
@@ -111,7 +113,7 @@ private[concurrent] final object Promise {
     final def this() = this(Noop: AnyRef)
 
     /**
-     * Returns the associaed `Future` with this `Promise`
+     * Returns the associated `Future` with this `Promise`
```

Comment:
should this say '`Future` associated with'?

```suggestion
     * Returns the `Future` associated with this `Promise`
```